test/e2e/serial/tests/configuration.go

```diff
@@ -257,11 +259,11 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 				return true, nil
 			}, 10*time.Minute, 30*time.Second).Should(BeTrue())
 
-			By(fmt.Sprintf("modifing the NUMAResourcesOperator ExporterImage filed to %q", nropTestCIImage))
+			By(fmt.Sprintf("modifing the NUMAResourcesOperator ExporterImage filed to %q", serialconfig.NropTestCIImage))
 			err = fxt.Client.Get(context.TODO(), client.ObjectKeyFromObject(nroOperObj), nroOperObj)
```

Comment:
can we use serialconfig.Config NROOperObj and NROSchedObj instead?

Reply:
we can evaluate (in a later PR) but in this case the test wants to alter the config mutating the objects, thus getting a local copy seems safer